packages/govern-console/src/containers/CreateDao/CreateDaoCollateral.tsx

```diff
@@ -0,0 +1,312 @@
+              }}
+            />{' '}
+            Any Address
+          </div>
+          {isAnyAddress ? (
+            <Info
+              mode={'warning'}
+              title={''}
+              style={{
+                marginTop: '20px',
+              }}
+            >
+              Please be aware that any address has permission to excute scheduels.
+            </Info>
+          ) : (
+            <div>
+              {executionAddressList.map((input, index) => (
+                <Split
+                  key={`exe${index}`}
+                  primary={
+                    <TextInput
+                      wide
+                      value={input}
+                      onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                        const newArr = [...executionAddressList];
+                        newArr[index] = e.target.value;
+                        updateCollaterals('executionAddressList', newArr);
+                      }}
+                    />
+                  }
+                  secondary={
+                    <Button
+                      mode={'secondary'}
+                      size={'large'}
+                      disabled={executionAddressList.length === 1}
+                      icon={<IconMinus />}
+                      onClick={() => {
+                        executionAddressList.splice(index, 1);
+                        updateCollaterals('executionAddressList', executionAddressList);
+                      }}
+                    />
+                  }
+                />
+              ))}
+              <Button
+                mode={'secondary'}
+                size={'large'}
+                label={'Add new address'}
+                icon={<IconPlus />}
+                display={'all'}
+                onClick={() =>
+                  updateCollaterals('executionAddressList', [...executionAddressList, ''])
+                }
+              />
+            </div>
+          )}
+
+          <Split
+            width={'100%'}
+            primary={
+              <Button
+                style={{ marginTop: spacing }}
+                size={'large'}
+                mode={'secondary'}
+                onClick={() => {
+                  setActiveStep(CreateDaoSteps.Config);
+                }}
+                icon={<IconArrowLeft />}
+                label={'back'}
+                display={'all'}
+              />
+            }
+            secondary={
+              <Button
+                style={{ marginTop: spacing, width: '100%' }}
+                size={'large'}
+                mode={'secondary'}
+                onClick={() => {
+                  setActiveStep(CreateDaoSteps.Review);
+                }}
+              >
+                Next Step
+              </Button>
+            }
+          />
+        </Box>
+      </GridItem>
+      <GridItem
+        gridRow={layoutName === 'large' ? '1' : undefined}
```

Comment:
does layoutName changes based on the device size?

Reply:
layoutName changes bases on windows width size, regardless of device size,
so for example: if the device is Ipad Pro, we probably get layout name 'medium' if the device is in portrait orientation, and 'large' if orientation is at landscape